### ISSUE TITLE
SRS Balance

### ISSUE THREAD

**Issue by TitanField** [HaloSpaceStation/HaloSpaceStation13#642]
The SRS hasn't been balanced in a very long time and it's quite annoying.

At the moment the SRS is a one shot kill on anything, during the most recent round (At the time of typing this) it became apparent that the SRS is really damn OP. Example: A marine was able to one shot an fully healed and shielded elite at point blank range and fired off 2-3 shots within a 1-2 second period that basically stunlocked them, they then brutally shot the hell out of said elite with the SRS.

I didn't follow the format since the format is set out for bugs while this is just a balance issue.

**Comment by Liam-Gallagher** [comment]
Ok, so a few corrections to the story as I was the marine who killed him. Me and another marine had engaged him, my fellow marine put a few rounds on target so he wasn't fully shielded. I ended up firing off a full mag from the SRS in close quarters, not a single round hit (as the SRS was nerfed to be nearly useless in close quarters). So I pulled back, reloaded, and shot him from range scoped in (He was completely stationary at the time so it was an easy kill). So for starters the whole point blank range part is wrong. Secondly my second shot didn't seem to connect (at least from my view), so I ran up on the down elite and shot him with about 1/4th a mag of an LMG not an SRS. So considering I misused the weapon entirely, wasted a full mag, and was still able to take out the elite, this is the fault of the elite, not the SRS. When you consider the elite's weapon would have nearly pain crit me or the other marine in only a burst or two, claiming the SRS is op, when it is only effective at range when scoped in, is ludicrous.
